core/network/src/config.rs

```diff
@@ -103,6 +103,11 @@ impl Roles {
 		self.intersects(Roles::FULL | Roles::AUTHORITY)
 	}
 
+	/// Does this role represents a client that does not participates in the consensus?
```

Comment:
Avoid questions in docs. But also, what does this mean? There's now no longer any reason that a full node with valid session keys for an authority should not be a validator. So if there is to be a CLI argument for this it should probably be `--force-disable-validating` to reflect the fact the full nodes should, by default, try to validate when they can.